themes/digital.gov/layouts/partials/core/head.html

```diff
@@ -12,6 +12,16 @@
   {{- end -}}
 {{- end -}}
 
+{{/* Set primary_image from topics/_index.md if not set on individual topic */}}
+{{ if eq .Type "topics" }}
+  {{ if .Params.primary_image }}
+    {{ $primary_image = .Params.primary_image }}
+  {{ else }}
+    {{ $topics_page := .Site.GetPage "topics" }}
+    {{ $primary_image = $topics_page.Params.primary_image }}
+  {{ end }}
+{{ end }}
+
```

Comment:
Is this necessary? The same param is getting checked on lines 5-12.

Reply:
I tried refactoring this code to check it once, but to set the correct default topics image and any topic specific override, we need to to check it within the `if eq .Type "topics"` "scope" to ensure the correct image is set.

Reply:
I understand now. Thanks for clearing that up.